### DISCUSSION TITLE
manage existing secret as mirror

### DISCUSSION THREAD

**Discussion by chidambaram27** [emberstack/kubernetes-reflector#448] [Ideas]
> Important: Reflector will skip any conflicting resource when creating auto-mirrors. If there is already a resource with the source's name in a namespace where an automatic mirror is to be created, that namespace is skipped and logged as a warning.

I saw this note, but it would be great to have another annotation to manage the existing secret as mirrors.

**Comment by winromulus** [comment]
@chidambaram27 I'm moving this to a discussion. I don't think this would be a good solution from a security point of view and it can break other k8s components.
